CONTRIBUTING.md

```diff
@@ -102,6 +102,7 @@ They are called [Processing Tags](#processing-tags) and [Attribute Tags](#attrib
 ---------------- | --- | ----
 `type`           | Yes | The type properties stores the format. It can currently be one of `gdb`, `shapefile`, `shapefile-polygon`, `csv`, `geojson`, or `xml` (for GML).
 `srs`            |     | Allows one to set a custom source srs. Currently only supported by `type:shapefile`, `type:shapefile-polygon`, and `type:csv`. Should be in the format of `EPSG:####` and can be any code supported by `ogr2ogr`. Modern shapefiles typically store their project in a `.prj` file. If this file exists, this tag should be omitted.
+`layer`          |     | The `gdb` source type allows multiple layers of geodata in a single input file. Use the `layer` tag to specify which of those layers to use. It can either be the string name of the layer or an integer index of the layer.
```

Comment:
This will work for any OGR source that supports layers — is there another we can use an example here, to clarify that it's not just GDB? Or will GDB practically be the only one that this works for?